src/content/resources/support.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Flutter support
+description: Where can you get support when developing with Flutter.
+---
+
+Welcome!
+
+There are many different ways to get support
+when developing with Flutter.
+As Flutter is an open source project,
+your options range from filing issues when you
+run into bugs in the software,
+to connecting with your local community,
+to browsing and identifying a list of commercial agencies
+and consultants who offer support services.
+
+### I want to report an issue with Flutter
+
+Visit how to [create a useful bug report](/resources/bug-reports).
+Note that filing bugs is _not_ an efficient way to get tech support.
+
+### I want to ask questions about how to use Flutter
+
+Your best bet is to consult
+[Stack Overflow](https://stackoverflow.com/questions/tagged/flutter),
+or [Discord](https://discord.com/invite/rflutterdev).
+While the community is active in these channels,
+there is no official SLA on responding to new posts
+or chats in these channels.
+
+### I want professional services support with Flutter
+
+Check out the
+[Flutter consultants]({{site.main-repo}}/consultants).
```

Comment:
nit: the Flutter consultants _directory_? or _list_?

seems like we might be missing a word after "consultants" ?